### PR TITLE
update micronaut to 2.4.0

### PR DESCRIPTION
**Pull request by atomfrede**
Hopefully this is just a drop in replacement.

### CODE REVIEW

**Comment by atomfrede** [comment]
Okay, hope is a bad companion

File: generators/server/templates/build.gradle.ejs [hunk 1]
```diff
@@ -586,9 +585,9 @@ wrapper {
 
 if (project.hasProperty("nodeInstall")) {
     node {
-        version = "${node_version}"
-        npmVersion = "${npm_version}"
-        yarnVersion = "${yarn_version}"
+        version = "$<%= NODE_VERSION %>"
```
Comment (JasonTypesCodes) [inline generators/server/templates/build.gradle.ejs:588]: It looks like the issue might be this `$`

Comment (JasonTypesCodes) [reply generators/server/templates/build.gradle.ejs:588]: Or at least... "an" issue ;)

Comment (atomfrede) [reply generators/server/templates/build.gradle.ejs:588]: :facepalm:

**Comment by atomfrede** [comment]
It seems something related to `jdbcOperations` has changed. The execution of the prepared statement is failing, not sure why yet.

**Comment by JasonTypesCodes** [comment]
Superseded by #236

**Pull request closed without merging**